### ISSUE TITLE
BULK ACTION Send Confirm Opt in Email doesn't work with "Select All"

### ISSUE THREAD

**Issue by gody01** [SuiteCRM/SuiteCRM#5882]
#### Issue
If Contacts (or any other Person) is selected with "Select All" options in List View, bulk action "Send Confirm Opt In Email" does not work.

Error returned: 1 email address do not have a valid id. 

#### Expected Behavior
Select Contacts with "Select All", bulk action "Send Confirm Opt In Email" SuiteCRM should put coresponding Email adresses to Email Queue (like it does, if we select entires on single page.

#### Actual Behavior
When we select Contacts in List View with "Select All", bulk action "Send Confirm Opt In Email" returns error message: 1 email address do not have a valid id. 

#### Possible Fix
Does not exist. 

#### Steps to Reproduce
1. Sales/Contacts
2. Filter desired Contacts
3. Select contacts with "Select All"
4. Select BULK ACTION/Send Confirm Opt In Email

#### Context
We can not send ConfirmOptIn Emails

#### Your Environment
* SuiteCRM Version used: 7.10.4
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox/Chrome/...
* Environment name and version (e.g. MySQL, PHP 7): latest distribution versions.
* Operating System and version (e.g Ubuntu 16.04): OpenSuSE Leap 42.3, CentOS 7, ...

**Comment by gody01** [comment]
One new observation (and accidental workaround).
If I select one page items on ListView (Contacts), then click on next page, select items there, click next page etc..., then click on  "Bulk Action/Send Confirm Opt In Email" items selected on all pages are sent to Queue.

**Comment by Kristianostman** [comment]
Same bug remain in 7.10.5. My workaround was like your but I did also set the list length to 500 instead of default 200 in System Settings to avoid too many clicks...

**Comment by dursuntosun** [comment]
Yes, when i try to use bulk action for confirm opt in email, it doesn't work. 
Error message: 1 email address do not have a valid id.

**Comment by lukeonorato** [comment]
Bulk actions errors are common, i've been trying fix an error likely yours, in my case, always when i try to execute a bulk action with all the records selected in any module, i get this message error "Selecting all records is not available for this action. Please select individual records." I'm not founding any solution yet.
